### ISSUE TITLE
Display of frequency in input line “Source” 0.5.0

### ISSUE THREAD

**Issue by ah1102** [DSheirer/sdrtrunk#859]
When assembling the program from source codes with the latest fixes, a problem was found with displaying the frequency in the “Source” input line. Maybe something is broken? I use to build JDK 14.0.1
![Снимок экрана 2020-06-11 в 19 45 17](https://user-images.githubusercontent.com/58341369/84416765-51863380-ac1d-11ea-8aae-61ff12f0fc27.png)

In the author's release - "0.5.0 alpha 1" there is no such problem.
![Снимок экрана 2020-06-11 в 19 44 36](https://user-images.githubusercontent.com/58341369/84416751-4af7bc00-ac1d-11ea-8a04-214da31577fa.png)

**Comment by DSheirer** [comment]
Nick,
I just updated the gradle build script to use a newer (13.0.2) set of JavaFX libraries.  Can you do a git pull and build and see if that clears the font rendering issues you're seeing?

**Comment by ah1102** [comment]
Yes

**Comment by ah1102** [comment]
It seems to me that he has become a little better or remained unchanged.

![Снимок экрана 2020-06-12 в 12 45 37](https://user-images.githubusercontent.com/58341369/84489731-cd7a8d00-acaa-11ea-8c04-508a2feff684.png)

**Comment by DSheirer** [comment]
When you click the 456.0 MHz line in the picture above, is the frequency value loading into the Source editor?

**Comment by ah1102** [comment]
No, in the program that I compiled from this code - https://github.com/DSheirer/sdrtrunk/releases/tag/v0.5.0-alpha.1

In your assembly sdr-trunk-osx-x86_64-v0.5.0-alpha1.zip everything is fine!

**Comment by DSheirer** [comment]
Can you build from the master branch and test tha? 

 (not the v0.5.0-alpha.1 tag)

**Comment by ah1102** [comment]
Yes, I did "git pull", as you requested from the main branch https://github.com/DSheirer/sdrtrunk.git and posted you a screenshot where the font is not visible to change.

Also, there is no frequency display in the "Source" line:

`When you click the 456.0 MHz line in the picture above, is the frequency value loading into the Source editor?`

**Comment by DSheirer** [comment]
Can you post the playlist file that you're using for this test.  I want to see if there is anything odd about the channel where the frequency value is not displaying in the source frequency field.

**Comment by ah1102** [comment]
Ok

[test.xml.zip](https://github.com/DSheirer/sdrtrunk/files/4770160/test.xml.zip)

**Comment by DSheirer** [comment]
The playlist looks good.  I have to go to work ... I'll try to look into this more over the weekend.

**Comment by ah1102** [comment]
Ok, thanks for the help! Have a nice day!!

**Comment by ah1102** [comment]
I found that in the program that I compiled from the source code, fewer files:
![Снимок экрана 2020-06-12 в 13 02 00](https://user-images.githubusercontent.com/58341369/84493207-12ed8900-acb0-11ea-9225-49dbacf7a06a.png)

Here is your folder and there are more files in it:
![Снимок экрана 2020-06-12 в 13 02 09](https://user-images.githubusercontent.com/58341369/84493283-2e589400-acb0-11ea-8f0f-19b89ce505c5.png)

**Comment by DSheirer** [comment]
The difference is you're building and using your local JDK14.  

The release version has its own (internal) JDK 14 version.

**Comment by ah1102** [comment]
Maybe I'm building the program incorrectly? I recorded a video of the process https://youtu.be/rmfWq9lShcc

**Comment by ah1102** [comment]
I downloaded your release "Version 0.5.0 Alpha 2" and everything is fine in it, except for rendering fonts. The frequency in the line is displayed. What could be wrong with the assembly on my computer? Maybe somewhere you need to run the sudo command?

**Comment by ah1102** [comment]
I changed the localization of the system interface from Russian to English as the main one. And now, the numbers have appeared! Surprisingly, when you return Russian as your main language, they are no longer displayed.
I think that with other regional localizations this may also be the case.

**Comment by ah1102** [comment]
For example, I tried to install Finnish as the macOS system language, and the numbers stopped showing.

Suomi
![Näyttökuva 2020-6-18 kello 19 54 01](https://user-images.githubusercontent.com/58341369/85051219-ea79f900-b19f-11ea-9198-acfafb45e08e.png)

English
![Screenshot 2020-06-18 at 19 56 44](https://user-images.githubusercontent.com/58341369/85051289-01205000-b1a0-11ea-8dd4-a338fdc5c8b9.png)

**Comment by ah1102** [comment]
Danny, are you registered with the Java bug tracker? Can you create a ticket?
